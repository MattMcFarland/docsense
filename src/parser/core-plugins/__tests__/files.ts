import filePlugin from '../file';
import loadPlugin from './utils/loadPlugin';

describe('Core Plugin: files', () => {
  test('matches snapshot', async () => {
    const runTest = loadPlugin({}, '__TEST__');
    const state = await runTest(filePlugin, 'foo()');
    const actual = state.file_collection;
    expect(actual).toMatchSnapshot();
  });
});
