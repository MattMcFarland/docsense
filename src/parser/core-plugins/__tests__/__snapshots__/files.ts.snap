// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Core Plugin: files matches snapshot 1`] = `
Array [
  Object {
    "base": "__TEST__",
    "dir": "",
    "ext": "",
    "id": "L19fVEVTVF9f",
    "isIndex": false,
    "kind": "Other",
    "name": "__TEST__",
    "path": "__TEST__",
  },
]
`;
